functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
 
-If you were to instead ask an archaeologist what should be done with the old data, the answer would be "hopefully, it’s just buried underneath".
+If you were to instead ask an archaeologist what should be done with the old data, the answer would be "hopefully, just bury it underneath".
 
-(Disclaimer: my understanding of a typical archaeologist is based on a few museum visits, several wikipedia articles, and watching the entire Indiana Jones series).
+(Disclaimer: My understanding of the views of a typical archaeologist is based on visiting a few museums, reading several Wikipedia articles, and watching the entire Indiana Jones series.)
 
-### From archaeology to databases
+### From Archaeology to Databases
 
-If we were to ask our friendly archaeologist to design a database, we might expect the requirements to reflect what would be found at an *excavation site:*
+If we were to ask our friendly archaeologist to design a database, we might expect the requirements to reflect what would be found at an excavation site:
 
-* All data is found and cataloged at the site
-* Digging deeper will expose the state of things in times past 
-* Artifacts found at the same layer are from the same period
-* Each artifact will consist of state that it accumulated in different periods 
+* All data is found and catalogued at the site.
+* Digging deeper will expose the state of things in times past.
+* Artifacts found at the same layer are from the same period.
+* Each artifact will consist of state that it accumulated in different periods.
 
-For example, a wall may have Roman symbols drawn on it at on one layer, and in a lower layer there may be Greek symbols. Both these observations are recorded as part of the wall's state.
+For example, a wall may have Roman symbols on it on one layer, and in a lower layer there may be Greek symbols. Both these observations are recorded as part of the wall's state.
 
 This analogy is visualized in Figure 1:
 
-* The entire circle is the excavation site
-* Each ring is a _layer_ (here numbered from 0 to 4) 
-* Each slice is a labeled artifact (‘a’ through ‘e’)
-* Each artifact has a ‘color’ attribute (where white means that no update was made)
-* Black arrows denote a change in color between layers (e.g., from c.color @t2 to c.color @t0)
-* Light blue arrows are arbitrary relationships of interest between entities (e.g., from ‘b’ to ‘d’)
+* The entire circle is the excavation site.
+* Each ring is a _layer_ (here numbered from 0 to 4).
+* Each slice is a labeled artifact (‘a’ through ‘e’).
+* Each artifact has a ‘color’ attribute (where white means that no update was made).

```

Comment:
The book will be printed in black and white (greyscale) -- so these figures and the relevant text need to be changed.